### PR TITLE
added URL placeholder

### LINKED ISSUE #193: Add details to 'Edit Profile' Form
1. Inputs Fields like 'Homepage' and 'Blog' require user to enter a URL. However, user comes to know this only when he/she has press the submit button once with some incorrect entry. It would be better if we could explicitly tell the user to enter a URL, by adding a message like - "Enter URL" or "eg http://myblog.com" to the label.
2. We should mention the max length in the label which the user can enter beside the 'First Name' and 'Last Name' fields.
@ana-balica  I would like to work on this.

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**
URL placeholder added at blog url and home page url so that user come to know that url has to be entered at that place. Fixes #193.

### CODE REVIEW

File: systers_portal/users/forms.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ def save(self, *args, **kwargs):
 
 class SystersUserForm(forms.ModelForm):
     """Form for SystersUser model"""
+    blog_url = forms.URLField(initial="http://",widget=forms.TextInput(attrs={'placeholder': 'Enter URL'}))
+    homepage_url = forms.URLField(initial="http://",widget=forms.TextInput(attrs={'placeholder': 'Enter URL'}))
```
Comment (tapasweni-pathak) [inline systers_portal/users/forms.py]: You also need to adapt this in test and model.

Comment (Himanshi-Khandelwal) [reply systers_portal/users/forms.py]: @tapasweni-pathak in tests the url is mentioned directly . So what to add there? Also what all to do in models.py?

**Comment by Himanshi-Khandelwal** [comment]
@tapasweni-pathak  Please tell me what to include in models.py

File: systers_portal/users/forms.py (hunk 1 above)
Comment (tapasweni-pathak) [reply systers_portal/users/forms.py]: can you please check this with `makemigrations` and `migrate`.

**Pull request closed without merging**